topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+
+#### Genome binning challenge
+
+Genome binning  clusters or classifies sequences or contigs into bins representing genomes. 
+
+It takes as input the output of assembly. Two types of short read assemblies were given as inputs for CAMI2 [[2]](https://www.biorxiv.org/content/10.1101/2021.07.12.451567v1): MEGAHIT [[10]](https://www.biorxiv.org/lookup/external-ref?access_num=25609793&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2021%2F07%2F12%2F2021.07.12.451567.atom) assembly and Gold Standard Assembly. Both types for strain madness, marine, and plant-associated datasets, so 6 possible datasets in total.
+
+Participants submitted 95 results and 10 binning tools (overall, 18 binner versions). The results were evaluated based on .
+
+ 
+
+
+
+* the average purity of bins, 
+    * The average purity is the fraction of correctly assigned base pairs for all assignments to a given bin averaged over all predicted genome bins, where unmapped genomes are not considered. 
+    * This value reflects how trustworthy the bin assignments are on average. 
+* completeness of genomes recovered,
+* Adjusted Rand Index which quantifies binning performance for the overall dataset.
+    * According to the [paper](https://www.biorxiv.org/lookup/google-scholar?link_type=googlescholar&gs_type=article&author[0]=F.+Meyer&title=AMBER:+Assessment+of+Metagenome+BinnERs&publication_year=2018&journal=Gigascience&volume=7) [[11]](https://academic.oup.com/gigascience/article/7/6/giy069/5034950?login=true), genome binners generate groups or clusters of reads and contigs for a given dataset. Instead of calculating performance metrics established with a bin-to-genome mapping, the quality of a clustering can be evaluated by measuring the similarity between the obtained and correct cluster partitions of the dataset, corresponding here to the predicted genome bins and the gold standard contig or read genome assignments, respectively. Two contigs or reads of the same genome that are placed in the same predicted genome bin are considered true positives TP. Two contigs or reads of different genomes that are placed in different bins are considered true negatives TN. The Rand index ranges from 0 to 1 and is the number of true pairs, TP + TN, divided by the total number of pairs. However, for a random clustering of the dataset, the Rand index would be larger than 0. The adjusted Rand index (ARI) corrects for this by subtracting the expected value for the Rand index and normalising the resulting value, such that the values still range from 0 to 1.
```

Comment:
I won't comment on every single reference issue, but, the comments above apply to all of the references throughout this tutorial